frontend/src/features/mapSideBar/SidebarContents/subforms/ParcelIdentificationForm.tsx

```diff
@@ -90,50 +87,73 @@ export const ParcelIdentificationForm: React.FC<IIdentificationProps> = ({
           <h5>Search for Parcel</h5>
         </Col>
         <Col md={6}>
-          <PidPinForm
-            handlePidChange={handlePidChange}
-            handlePinChange={handlePinChange}
-            nameSpace={nameSpace}
-          />
-          <CenteredSearchButton
-            onClick={(e: any) => {
-              e.preventDefault();
-              let newValues = {
-                ...formikProps.values,
-              };
-              newValues = setIn(newValues, nameSpace, getInitialValues());
-
-              formikProps.resetForm({
-                values: newValues,
-              });
-
-              if (pid?.length > 0) {
-                handlePidChange(pid, nameSpace);
-              } else if (pin?.length > 0) {
-                handlePinChange(pin, nameSpace);
-              } else if (geocoderResponse) {
-                handleGeocoderChanges(geocoderResponse, nameSpace);
-              }
-            }}
-          />
-          <Form.Row className="justify-content-end">
+          <Form.Row>
+            <Label>PID</Label>
+            <Input
+              displayErrorTooltips
+              className="input-small"
+              disabled={false}
+              pattern={RegExp(/^[\d\- ]*$/)}
+              onBlurFormatter={(pid: string) => {
+                if (pid?.length > 0) {
+                  return pid.replace(pid, pidFormatter(pid));
+                }
+                return '';
+              }}
+              field={withNameSpace('searchPid')}
+            />
+            <SearchButton
+              onClick={(e: any) => {
+                e.preventDefault();
+                handlePidChange(getIn(formikProps.values, withNameSpace('searchPid')), nameSpace);
+              }}
+            />
+          </Form.Row>
+          <Form.Row>
+            <Label>PIN</Label>
+            <FastInput
+              formikProps={formikProps}
```

Comment:
I wonder if we could use a `Context` for our Formik controls.  We're always passing down the props anyway.

Reply:
formik already puts all of its data into a context provider that is internal to formik. You can get access to that via useFormikContext().

This is required because I'm using the old style of memo checking provided in FastInput. Could log tech debt to remove this prop, but I'm not convinced we would see any value other then one less prop on the Fast* variants of fields.